<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="5262" started-at="2014-10-07T06:11:47Z" finished-at="2014-10-07T06:11:52Z">
    <groups>
      <group name="android">
        <method signature="testandroid.testandroidcase()[pri:0, instance:com.appium.test.testandroid@3ca83695]" name="testandroidcase" class="com.appium.test.testandroid"/>
      </group> <!-- android -->
    </groups>
    <test name="Default test" duration-ms="5262" started-at="2014-10-07T06:11:47Z" finished-at="2014-10-07T06:11:52Z">
      <class name="com.appium.test.testandroid">
        <test-method status="PASS" signature="getDriverInstance()[pri:0, instance:com.appium.test.testandroid@3ca83695]" name="getDriverInstance" is-config="true" duration-ms="6971" started-at="2014-10-07T11:41:40Z" finished-at="2014-10-07T11:41:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getDriverInstance -->
        <test-method status="FAIL" signature="testandroidcase()[pri:0, instance:com.appium.test.testandroid@3ca83695]" name="testandroidcase" duration-ms="5144" started-at="2014-10-07T11:41:47Z" finished-at="2014-10-07T11:41:52Z">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[Cannot find elements when the XPath expression is null.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: Cannot find elements when the XPath expression is null.
	at org.openqa.selenium.By.xpath(By.java:112)
	at com.appium.pages.HomePage.<init>(HomePage.java:32)
	at com.appium.test.testandroid.testandroidcase(testandroid.java:15)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testandroidcase -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.appium.test.testandroid@3ca83695]" name="tearDown" is-config="true" duration-ms="3478" started-at="2014-10-07T11:41:52Z" finished-at="2014-10-07T11:41:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.appium.test.testandroid -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
